web/slashes/scrapers.py

```diff
@@ -0,0 +1,218 @@
+import os
+import re
+import bs4
+import html
+from typing import List
+from flask import current_app, request
+
+from . import commands
+from ..executor import executor
+from ..utils import ForgetfulDict
+from .. import http, slack, formatting, utils
+
+
+class WebScrapingCommandHandler(commands.CommandHandler):
+    def __init__(self, *, relink_timeout):
+        super().__init__()
+
+        self.recently_linked_cache = ForgetfulDict(memory_time=relink_timeout)
+
+    def handle(self):
+        channel = request.form.get("channel_id")
+
+        args = []
+        skipped_args = []
+        requested_args = html.unescape(request.form.get("text")).split(" ")
+
+        for arg in requested_args:
+            if (arg.upper(), channel) not in self.recently_linked_cache:
+                self.recently_linked_cache[(arg.upper(), channel)] = True
```

Comment:
```suggestion
            key = (arg.upper(), channel)
            if key not in self.recently_linked_cache:
                self.recently_linked_cache[key] = True
```

Reply:
I love that we only have to make the change here!

Reply:
61827a8